src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigCol {
```

Comment:
The class name 'FigCol' suggests a verb-like action which is not recommended for class names. Consider using a noun that better represents the class's purpose, such as 'ColoredFigure'.